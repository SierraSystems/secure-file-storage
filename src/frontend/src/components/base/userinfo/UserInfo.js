import React from "react";

export default function UserInfo() {
  return (
    <div className="spacing">
      <p className="head">
        <b>Name: </b>
        {sessionStorage.getItem("displayName")}
      </p>
      <p className="head">
        <b>Birthdate: </b>
        {sessionStorage.getItem("birthdate")}
      </p>
      <p className="head">
        <b>Gender: </b>
        {sessionStorage.getItem("gender")}
      </p>
      <p className="head">
        <b>Address: </b>
        {sessionStorage.getItem("address")}
      </p>
    </div>
  );
}
