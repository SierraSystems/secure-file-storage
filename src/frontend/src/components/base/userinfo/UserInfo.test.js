import React from "react";
import UserInfo from "./UserInfo";
import testBasicSnapshot from "../../../TestHelper";

describe("UserInfo", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<UserInfo />);
  });
});
