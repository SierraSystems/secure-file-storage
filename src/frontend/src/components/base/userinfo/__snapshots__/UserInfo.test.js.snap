// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo Component renders as expected 1`] = `
<div
  className="spacing"
>
  <p
    className="head"
  >
    <b>
      Name: 
    </b>
  </p>
  <p
    className="head"
  >
    <b>
      Birthdate: 
    </b>
  </p>
  <p
    className="head"
  >
    <b>
      Gender: 
    </b>
  </p>
  <p
    className="head"
  >
    <b>
      Address: 
    </b>
  </p>
</div>
`;
